lib/helpers/is-route-template.js

```diff
@@ -19,6 +19,6 @@ module.exports = function isRouteTemplate(moduleName) {
   var isPartial = baseName.indexOf('-') === 0;
 
   return (
-    !isPartial && moduleName.indexOf('components/') === -1 // classic component
+    !isPartial && moduleName.indexOf('/templates/components/') === -1 // classic component
```

Comment:
Hmm, with the upcoming changes for co-location (emberjs/rfcs#481) we won't be using `app/templates/components` any longer.

Specifically, these are both valid component template locations:

```js
app/components/foo.hbs
app/components/foo/index.hbs
```

Reply:
So could we do

```es6
!isPartial && moduleName.indexOf('<appname>/templates/components/') === -1 && moduleName.indexOf('<appname>/components/') === -1
```

instead then?

Reply:
@marcoow @rwjblue What about just checking for the `/components/` path?:
```js
!isPartial && !moduleName.includes('/components/')
```

Reply:
Well, reading the original issue we cannot do just the `/components` string given that example route.

Reply:
`<appname>` would be `app` in most tests here, yes - in reality it's the app's actual module name, e.g. `my-app`. I updated this to support co-located component templates as well.